scipy/stats/_multicomp.py

```diff
@@ -26,11 +26,48 @@
 class DunnettResult:
     statistic: np.ndarray
     pvalue: np.ndarray
+    _alternative: Literal['two-sided', 'less', 'greater']
     _rho: np.ndarray
     _df: int
     _std: float
     _observations_mean: np.ndarray
     _control_mean: np.ndarray
+    _ci: ConfidenceInterval | None = None
+    _ci_cl: DecimalNumber | None = None
+
+    def __str__(self):
+        # Note: `__str__` prints the confidence intervals from the most
+        # recent call to `confidence_interval`. If it has not been called,
+        # it will be called with the default CL of .95.
+        if self._ci is None:
+            self.confidence_interval(confidence_level=.95)
+        s = (
+            "Dunnett's test"
+            f" ({self._ci_cl*100:.1f}% Confidence Interval)\n"
+            "Comparison               Statistic  p-value  Lower CI  Upper CI\n"
+        )
+        for i in range(self.pvalue.shape[0]):
+            s += (f" (Sample {i} - Control) {self.statistic[i]:>10.3f}"
+                  f"{self.pvalue[i]:>10.3f}"
+                  f"{self._ci.low[i]:>10.3f}"
+                  f"{self._ci.high[i]:>10.3f}\n")
+
+        if self._alternative == 'less':
+            s += (
+                "\nOne-sided alternative (less): sample i's mean exceed "
+                "the control's mean by an amount at least Lower CI"
```

Comment:
I'm confused.

- By "exceed" did you mean "exceeds", since  "sample i's mean" is singular?
- This is a statement about the mean of the samples passed into the function. This statement may be true or false, depending on what the sample means actually are, but the statement is printed without a condition on the sample means.
- I think this is intending to say something about the true values of the population means underlying the samples, but it does not.

Reply:
I thought it was ok as in the article Dunnett seemed to used the same formulation for both cases